crates/ty_ide/src/inlay_hints.rs

```diff
@@ -641,6 +867,81 @@ mod tests {
         y[: tuple[Literal[1], Literal["abc"]]] = x
         z[: tuple[int, str]] = (i(1), s('abc'))
         w[: tuple[int, str]] = z
+
+        ---------------------------------------------
+        info[inlay-hint-location]: Inlay Hint Target
+           --> stdlib/builtins.pyi:348:7
+            |
+        347 | @disjoint_base
+        348 | class int:
+            |       ^^^
+        349 |     """int([x]) -> integer
+        350 |     int(x, base=10) -> integer
+            |
+        info: Source
+          --> main.py:9:2
+           |
+         7 | x = (1, 'abc')
+         8 | y = x
+         9 | z = (i(1), s('abc'))
```

Comment:
I find the tests here confusing where there's no inlay. Aren't those inlays that are hidden? Why do we render the inlay hint targets for those? I think we should omit those from the tests.